packages/components/src/placeholder/index.js

```diff
@@ -15,9 +15,9 @@ import Dashicon from '../dashicon';
 * @param  {Object} props The component props.
 * @return {Object}       The rendered placeholder.
 */
-function Placeholder( { icon, children, label, instructions, className, notices, preview, ...additionalProps } ) {
-	const classes = classnames( 'components-placeholder', className );
-
+function Placeholder( { icon, children, label, instructions, className, notices, preview, isColumnLayout, ...additionalProps } ) {
+	const classes = classnames( 'components-placeholder', className, );
```

Comment:
nit: This trailing comma isn't necessary.

```suggestion
	const classes = classnames( 'components-placeholder', className );
```